include/QCefView.h

```diff
@@ -59,6 +59,17 @@ class QCEFVIEW_EXPORT QCefView : public QWidget
   };
   Q_ENUM(CefWindowOpenDisposition)
 
+  /// <summary>
+  /// Represents the CEF javascript dialog type
+  /// </summary>
+  enum CefJsDialogType
+  {
+    JSDIALOGTYPE_ALERT = 0,
```

Comment:
can you rename the enum value to 
CefJsDialogTypeAlert = 0,
CefJsDialogTypeConfirm,
CefJsDialogTypePrompt